src/Runner.Worker/Handlers/StepHost.cs

```diff
@@ -264,14 +300,20 @@ await containerHookManager.RunScriptStepAsync(context,
 
         private string CheckPlatformForAlpineContainer(IExecutionContext executionContext, string preferredVersion)
         {
+            // Handle ARM32 architecture specifically for node24
+            preferredVersion = NodeCompatibilityChecker.CheckNodeVersionForLinusArm32(executionContext, preferredVersion);
+
             string nodeExternal = preferredVersion;
+
```

Comment:
```suggestion
string nodeExternal = NodeCompatibilityChecker.CheckNodeVersionForLinusArm32(executionContext, preferredVersion);
```

Reply:
thanks, I've done something similar now with the tuple return